app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -117,7 +117,7 @@ class LoginActivity : AppCompatActivity(), ILoginView {
     }
 
     fun signUp() {
-        sign_up.setOnClickListener { startActivity(Intent(this@LoginActivity, SignUpActivity::class.java)) }
+        login_activity_sign_up.setOnClickListener { startActivity(Intent(this@LoginActivity, SignUpActivity::class.java)) }
```

Comment:
Unrelated

Reply:
@iamareebjamal As the UI tests were failing that time and on going through the logs what I understood that time was that there was a trouble differentiating between the id "sign_up" which is present in both the Login and SignUp activity. Hence, I tried changing it for the Login Activity tests and it did pass the UI tests in Android studio. But, now I see that this doesn't turn up, there are some other issues. I will change this back to the original, as I have done in one of the other PRs.